modules/op-release-notes-1-20-2.adoc

```diff
@@ -0,0 +1,50 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-20.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-20-2_{context}"]
+= Release notes for {pipelines-title} 1.20.2
+
+With this update, {pipelines-title} General Availability (GA) 1.20.2 is available on {OCP} 4.16 and later versions.
+
+[id="enhancements-1-20-2_{context}"]
+== Enhancements
+
+.Operator
+
+Custom Hub catalog configuration fix::
+Before this update, the custom hub catalog name and URL for PAC were not retained during upgrade from 1.19.3 due to missing data persistence. As a result, user settings for custom hub catalog name and URL were lost after upgrading to 1.20.1, causing inconsistency in {pac} configuration. With this release, custom hub catalog settings are retained after upgrade to 1.20.2.
+
+[id="fixed-issues-1-20-2_{context}"]
+== Fixed issues
+
+.Operator 
+
+Custom hub catalog configuration is preserved during conversion::
+Before this update, the `Tekton` Operator removed the `catalog-{INDEX}-type` field during conversion, which caused the loss of custom hub catalog types. As a result, custom hub catalog configurations were broken, preventing users from configuring custom Tekton Hub, Artifact Hub, or other custom hubs through the `TektonConfig` custom resource (CR). With this update, the `Tekton` Operator preserves the `catalog-{INDEX}-type` field in its config map. As a result, you can now configure custom hub catalogs using the `TektonConfig` CR.
+
+.User interface
+
+Pagination works correctly for archived PipelineRun::
+With this update, pagination functions correctly when the `data_source` filter is set to `archived`. In the previous release, scrolling failed to load additional `PipelineRun` results because the UI client expected the `nextPageToken` field, while the API returned `next_page_token`. This mismatch prevented the `onScroll` callback from creating the next page request, causing pagination to stop after the first page. The UI has been updated to support `next_page_token`, and additional results load as expected when scrolling.
+
+.Tekton Config
```

Comment:
this should be Tekton Results, replace Tekton config with Tekton results

Reply:
Fixed.